hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/model/JpaArtifact.java

```diff
@@ -53,10 +53,19 @@
     @JoinColumn(name = "software_module", nullable = false, updatable = false, foreignKey = @ForeignKey(value = ConstraintMode.CONSTRAINT, name = "fk_assigned_sm"))
     private JpaSoftwareModule softwareModule;
 
+    @Column(name = "sha1_hash", length = 40, nullable = true)
```

Comment:
We never do them execept if we want to do a validation that is not supported by the database